Dokumentasjon/V1/Meldinger/metadatakatalog/README.md

```diff
@@ -82,4 +82,49 @@ Skal ikke kunne endres
 Ulike klassifikasjonssystemer innenfor samme arkivsystem kan inneholde en eller flere av de samme identifikasjonene. Identifikasjonen kan være rent nummerisk, men kan også være alfanumerisk og ha et logisk meningsinnhold. Merk at klasseID er identisk med begrepene ordningsverdi og arkivkode i Noark 4.
 
 ---
+### `kode` - FIKS-ARKIV-001
+#### Definisjon
+Kode er basen for felter som f.eks. dokumenttype som er av typen kode. Disse har da verdier fra kodelister. 
+Kode består av kode og beskrivelse. 
+Elementet kode er en kort identifkator mens beskrivelse kan være en lengre tekst som beskriver koden. F.eks. kode=F og beskrivelse=Faktura
+
+#### Arkivenhet
+
+#### Kilde
+
+#### Arv
+
+#### Betingelser
+
+#### Kommentarer
+Elementet kode er den som er påkrevd og er typisk en forkortelse for hva den representerer.
+
+---
+### `dokumenttype` - M083
+#### Definisjon
+Navn på type dokument
+
+#### Arkivenhet
+_dokumentbeskrivelse_ 
+
+#### Kilde
+Registreres automatisk av systemet eller manuelt
+
+#### Arv
+Kode
+
+#### Betingelser
+Ingen obligatoriske typer. Aktuelle kode/beskrivelser kan f.eks. være:
+
+| kode | beskrivelse      |
```

Comment:
Hent liste fra arkivlett: https://www.ks.no/contentassets/3d554878a07a4eb58338f8bafb956925/ArkivLett.pdf

Reply:
Ja, jeg fant ut at det var andre verdier vi hadde gått for i kodelistene vi laget for .NET utgaven av Fiks-Arkiv. Har oppdatert nå 👍